tic2.js

```diff
@@ -24,28 +24,24 @@ let printGrid = function () {
 }
 
 let playerTurn = async function () {
-    let input = await ask('\n> ');
-    if (input < 1 || input > 9) {
-        console.log("\nHey now Player " + player.turn + "! That's not a valid input. Please enter a number between 1 and 9!");
-        printGrid();
-        playerTurn();
-    }
-    else if (cells[parseInt(input) - 1] == 'X' || cells[parseInt(input) - 1] == 'O') {
-        console.log("\nSorry Player " + player.turn + "! That space has been claimed! Try again.");
+    let input = parseInt(await ask('\n> '));
+    if (!cells.includes(input)) {
```

Comment:
so... what happens if the user types 'X' ?

Reply:
yeah, github was confused ... i made that comment on the version that *didn't* parseInt... 

but also, you have two different files here (tic1 and tic2) which also confused me

Reply:
> so... what happens if the user types 'X' ?

If the user types 'X', parseInt will return NaN, which doesn't appear in the 'cells' array, so the input checker will reject it!

Reply:
Correct!